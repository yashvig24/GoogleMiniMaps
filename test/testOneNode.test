# test parser and shortestPath on one node graph

# build graph
LoadGraph oneNodeGraph oneNode.tsv

# check graph is built correctly
ListNodes oneNodeGraph
ListChildren oneNodeGraph Seattle

# check BFS
FindPath oneNodeGraph Seattle Seattle
FindPath oneNodeGraph Seattle Boston