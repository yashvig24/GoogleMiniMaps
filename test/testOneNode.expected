# test parser and shortestPath on one node graph

# build graph
loaded graph oneNodeGraph

# check graph is built correctly
oneNodeGraph contains: Seattle
the children of Seattle in oneNodeGraph are:

# check BFS
path from Seattle to Seattle:
total cost: 0.000
unknown character Boston